src/api/exporters/ezsLocals.js

```diff
@@ -84,7 +85,26 @@ export function JSONLDObject(data, feed) {
                 }
             }
         });
-        feed.send(output);
+
+        const transformCompleteFields = async (field) => {
```

Comment:
You should avoid declaring functions inside other functions as you won't be able to test them.